kolibri/auth/models.py

```diff
@@ -66,7 +66,12 @@ class FacilityDataset(models.Model):
     description = models.TextField(blank=True)
     location = models.CharField(max_length=200, blank=True)
 
-    allow_signups = models.BooleanField(default=True)
```

Comment:
I am thinking this is redundant to `learner_can_sign_up` below. Correct me if I am wrong @jamalex?